lib/vrl/tests/src/main.rs

```diff
@@ -40,6 +41,10 @@ pub struct Cmd {
     #[clap(short, long)]
     logging: bool,
 
+    /// When enabled, show run duration for each individual test.
+    #[clap(short, long)]
+    timings: bool,
```

Comment:
Yeah, these things tend to be introduced in PRs like these, as there's no "let's improve the test harness" work scheduled most of the time, but sometimes these things are added as development happens because they can give insights that were missing during development.

In this case, I added it because I noticed the test harness got "stuck" on certain tests, and it was hard to pinpoint which ones they were.

I could've pushed it into a follow-up PR, though, I agree. But alas, it's here now, and it's contained to a single file, so I suggest we keep it in.